src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAHelper.h

```diff
@@ -35,6 +35,7 @@
 #pragma once
 
 #include <OpenMS/CHEMISTRY/AASequence.h>
+#include <OpenMS/KERNEL/RangeManager.h>
```

Comment:
avoid includes (compile time cost) -- here you can get away with a forward declaration of `RangeMZ` etc, since they are simple classes.
Also in other headers.